README.md

```diff
@@ -1,39 +1,167 @@
-# Getting Started
-Welcome to getting started!
-The purpose of this repo is to get you started with git and GitHub
-so you can develop software in the Billinge Group.
-
-
-# Tutorial
-1. Read chapters 15 and 16 of "Effective Computation in Physics" by
-Scopatz and Huff (it is strongly recommended that you get a copy, but we do have
-
-
-Below is Exercise 1 of the tutorial.
-If you have any questions about this, or future exercises, please:
-1. Google a little, or look it up in Scopatz and Huff
-1. Ask a group member for help
-1. File an issue
-
-Exercise 1: forking, cloning, PR'ing
-1. Make a GitHub account, if you don't already have one.
-1. Make sure you have a working installation  of `git` on your local
-machine
-1. Fork this repo
-1. Clone your fork
-1. Checkout a new branch.  Give it a helpful name.
-1. Add yourself to the list of people who completed the training on a new line
-(see `finished.txt`)
-1. Push your changes to your fork's branch
-1. Put in a Pull Request into upstream master
-1. Request a review from your mentor
-
-Exercise 2: finishing up the old work
-1. After your changes are successfully reviewed, revised, and pulled in, it is
-time to clean up your stuff.
-1. Synchronize the master branch of your local clone with upstream master
-1. Delete the branch you used for the finished.txt update
-1. In a good old-fashioned email, write a brief account of everything you did to
-complete Exercise 1 and 2 and why you did it (i.e., what purpose did each step
-serve)
+# Git workflow
+
+This page is aimed to be a simplified version of github help. For complete workflow,
+please go to [full git documentation](https://guides.github.com/). Also Google and [stackoverflow](https://stackoverflow.com/) is incredibly helpful.
+
+Below is a code block for reference. It includes the most basic git commands you will
+use to complete the workflow introduced below. If it's your first time
+here, skip this block and finish reading the workflow doc first.
+```git
+git clone <remote url>  # clone a repo
+git remove add <name> <remote url>  # add a remote
```

Comment:
Now for these we can just have a section:
Some useful git commands in this workflow